code/game/objects/items/weapons/grenades/frag.dm

```diff
@@ -37,24 +39,24 @@
 
 	var/mob/living/carbon/human/H = target
 	if(!prob(embed_prob - ARMOUR_VALUE_TO_PERCENTAGE(H.getarmor(null, BOMB))))
-		to_chat(H, "<span class='warning'>Shrapnel bounces off your armor!</span>")
+		to_chat(H, "<span class='warning'>An explosive fragment bounces off your armor!</span>")
 		return
 	H.try_embed_object(new_possible_embed)
 
-/obj/item/projectile/bullet/shrapnel/on_range()
-	var/obj/item/we_missed = new embedded_type(get_turf(src)) // we missed, lets toss the shrapnel
+/obj/item/projectile/bullet/fragment/on_range()
+	var/obj/item/we_missed = new embedded_type(get_turf(src)) // we missed, lets toss the fragment
 	var/range = gaussian(4, 2)
 	if(range > 0)
 		var/atom/i_wasnt_aiming_for_the_truck = get_angle_target_turf(get_turf(src), Angle, range)
 		we_missed.throw_at(i_wasnt_aiming_for_the_truck, 16, 3)
 	return ..()
 
 /**
- * Shrapnel projectiles turn into this after trying to embed
+ * Fragment projectiles turn into this after trying to embed.
  */
-/obj/item/shrapnel
-	name = "shrapnel"
-	desc = "Metal shards at high velocity, a classic method of blowing your enemies up."
+/obj/item/fragment
+	name = "fragment"
+	desc = "Metal shards at high velocity, a classic method of blowing your enemies up. Often incorrectly referred to as \"shrapnel\"."
```

Comment:
Though the two are practically synonyms nowadays?

Reply:
> Though the two are practically synonyms nowadays?

To lay persons yes, but they do retain specific technical distinctions. A shrapnel bomb is essentially just a shotgun shell except with a much wider cone, or just omnidirectional. The key distinction is the projectiles are not made of the destroyed casing.

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/143041327/bb8963ac-4e2a-4f4e-8e7b-5bcfd314a788)

Reply:
Its still not clear in a game of 10000 items what an item called a "fragment" is. It may be more TECHNICALLY correct, but this is a game not a PHD dissertation. Items that would require explanation when their name could simply be changed is not a good design and will only lead to confusion amongst players.

Reply:
Fine. I changed it back.